---
layout: default
---

<div id="header" style="text-align: center;">
  <img src="{{ site.baseurl }}/docs/images/header.development.png" alt="Centered Image" style="margin-top: 20px;">
</div>

## Scientists Measure Gene Expression Changes Along the Internal Branches of Species Trees

<span class="dropcap">I</span>n a paper by scientists at the University of Lausanne and the Swiss Institute of Bioinformatics, [published in 2011 in the prominent journal Nature](https://doi.org/10.1038/nature10532), Brawand and colleagues investigated the transcription rate changes along branches of species trees build from contemporary samples of RNA-seq reads from nine mammalian species.

To initially improve the annotation of genes, the RNA-seq reads were mapped against the corresponding species' reference genomes. Because introns are separated from exons during normal pre-mRNA processing the reads mapped only to exons, which revealed where the gene splice sites are. This allowed for the identification of alternative splice sites, appearing as introns surrounded by exons of different expression rates, letting the authors make a more accurate description of the expressed genes.

The researchers then identified orthologous genes of the nine species and created species trees based on expression profiles. The total branch length of these trees varied among tissues, with testis showing the highest level of expression divergence among species. They also found that there is a limit to the accumulation of expression divergence between species. For example, humans are as different in their expression levels from marsupials as they are from chickens, despite having diverged 110 million years earlier from chickens than from marsupials.

To identify chromosomes that underwent significant expression changes in the past, the researchers compared the lengths of equivalent branches between expression trees for different chromosomes. They discovered a remarkable change in the expression of genes on the X chromosome compared to those on autosomal chromosomes (non-sex chromosomes). This change occurred on the evolutionary branch leading from the common ancestor of all mammals, including egg-laying monotremes like the platypus, to the common ancestor of therian mammals (marsupials and placental mammals).

This branch represents a critical point in mammalian evolution. It diverges from the branch leading to monotremes and moves towards therians, which give birth to live young. Interestingly, monotreme males have five X chromosomes, while therian males have only one. The researchers believe that the observed change in X-linked gene expression on the therian branch likely reflects this drastic reduction in the number of X chromosomes, which would affect the "dosage" or amount of genetic material from these chromosomes.[^1]

[^1]: Brawand, D., Soumillon, M., Necsulea, A. *et al.* [The Evolution of Gene Expression Levels in Mammalian Organs](https://doi.org/10.1038/nature10532). *Nature* __478__, 343-348 (2011). https://doi.org/10.1038/nature10532









